.cbmc-batch/jobs/aws_nospec_mask/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwinding-assertions;--unwind;1"
```

Comment:
Should we just remove `--unsigned-overflow-check` here?

Reply:
This would disable the unsigned overflow check for all code covered by this proof. However, the right thing would be to disable the unsigned overflow check only in the portion of the code that deliberately uses these overflows. To that end, we have the following pragmas

```
#pragma CPROVER check push
#pragma CPROVER check disable "unsigned-overflow"

// Code with known unsigned overflows

#pragma CPROVER check pop
```